scripts/generate_rust_analyzer.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+import sys
+import os
+from os import path
```

Comment:
Nowadays there is `pathlib` which is nicer, give it a look!

Reply:
That's fine, as mentioned above, feel free to leave it as it is.

AFAIU, it is easy to test a new version by comparing old vs. new output, right? So someone that wants to clean it up should not have a problem testing it.

Reply:
Yes, that should work for testing. I just restarted rust-analyzer which each new try